dist/jssip.js

```diff
@@ -598,6 +598,14 @@ module.exports = /*#__PURE__*/function () {
         // RFC 3261 13.2.2.4.
         this._route_set = message.getHeaders('record-route').reverse();
       }
+      var cseq = message.cseq ? parseInt(message.cseq, 10) : null;
```

Comment:
`message.cseq` is guaranteed to be there. `sanityCheck.js` use from `UA.js` ensures it.